res/TensorFlowLiteRecipes/ReverseSequence_000/test.recipe

```diff
@@ -0,0 +1,28 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 4 dim: 8 }
+}
+operand {
+  name: "seq_lengths"
+  type: INT32
+  shape { dim: 4 }
```

Comment:
set fill data, otherwise this recipe could generate invalid tflite file.
It will work for now, since we do not test behavior of generated files, but fill crash once interpreter can run this operator